packages/editor/src/components/default-block-appender/style.scss

```diff
@@ -1,14 +1,16 @@
 .editor-default-block-appender {
-	input[type="text"].editor-default-block-appender__content { // needs specificity
+	clear: both; // The appender doesn't scale well to sit next to floats, so clear them.
+
+	input[type="text"].editor-default-block-appender__content { // Needs specificity.
```

Comment:
I don't think this comment is helpful, explaining *why* it needs the specificity would be handy 😄